content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md

```diff
@@ -151,61 +169,72 @@ Edit your `validators.txt` file. This file is located in the same folder as your
 
-To connect to the NFT-Devnet, you must also forcefully enable the NFT feature, `XLS20`. Add or modify the `[features]` stanza of your config file as follows:
+<!-- MULTICODE_BLOCK_START -->
+
+_Testnet_
 
 ```
-[features]
-NonFungibleTokensV1
+# [features]
+# Delete or comment out. Don't force-enable features on Testnet.
 ```
 
-**Warning:** Do not use the `[features]` stanza when connecting to Mainnet or Testnet. Forcefully enabling different features than the rest of the network could cause your server to diverge from the network.
+_Devnet_
 
-## 4. Restart the server.
+```
+# [features]
+# Delete or comment out. Don't force-enable features on Devnet.
+```
 
-```sh
-$ sudo systemctl restart rippled
+_Mainnet_
+
+```
+# [features]
+# Delete or comment out. Don't force-enable features on Mainnet.
 ```
 
-## 5. Verify that your server syncs.
+_NFT-Devnet_
 
-It takes about 5 to 15 minutes to sync to the network after a restart. After your server is synced, the [server_info method][] shows the a `validated_ledger` object based on the network you are connected to.
+```
+[features]
+NonFungibleTokensV1
+```
 
-To verify that your `rippled` is connected to the XRP Testnet or Devnet, compare the results from your server to [a public server][public servers] on the Testnet or Devnet. The `seq` field of the `validated_ledger` object should be the same on both servers (possibly off by one or two, if it changed as you were checking).
+_AMM-Devnet_
 
-The following example shows how to check the latest validated ledger from the commandline:
+```
+[features]
+AMM
+```
 
-<!-- MULTICODE_BLOCK_START -->
+<!-- MULTICODE_BLOCK_END -->
 
-*Local Server*
+(On Mainnet, Testnet, and Devnet, no you)
 
-```sh
-rippled server_info | grep seq
-```
+**Warning:** Do not use the `[features]` stanza when connecting to Mainnet or Testnet. Forcefully enabling different features than the rest of the network could cause your server to diverge from the network.
 
-*Testnet*
+## 4. Restart the server.
 
 ```sh
-rippled --rpc_ip s.altnet.rippletest.net:51234 server_info | grep seq
+$ sudo systemctl restart rippled
 ```
 
-*Devnet*
+## 5. Verify that your server syncs.
 
-```sh
-rippled --rpc_ip s.devnet.rippletest.net:51234 server_info | grep seq
-```
+It takes about 5 to 15 minutes to sync to the network after a restart. After your server is synced, the [server_info method][] shows the a `validated_ledger` object based on the network you are connected to.
 
+To verify that your `rippled` is connected to the right network, compare the results from your server to [a public server][public servers] on the Testnet or Devnet. The `seq` field of the `validated_ledger` object should be the same on both servers (possibly off by one or two, if it changed as you were checking).
```

Comment:
```suggestion
To confirm that your `rippled` is connected to the right network, compare the results from your server to [a public server][public servers] on the Testnet or Devnet. The `seq` field of the `validated_ledger` object should be the same on both servers (possibly off by one or two, if it changed as you were checking).
```

`verify` seems too strong a word, since we're just looking at the `seq` number. If two new ledgers are launched at about the same time, or if there is a fork/divergence in a network, then this test could mislead.